gui/map.cc

```diff
@@ -89,13 +90,13 @@ Map::Map(QWidget* parent,
   connect(mclicker, &MarkerClicker::logTime, this, &Map::logTime);
 
   // We search the following locations:
-  // 1. In the file system in the same directory as the executable.
+  // 1. In the file system in the QStandardPaths::AppDataLocation directories.
   // 2. In the Qt resource system.  This is useful if the resource was compiled
   //    into the executable.
-  QString baseFile =  QApplication::applicationDirPath() + "/gmapbase.html";
+  QString baseFile = QStandardPaths::locate(QStandardPaths::AppDataLocation, "gmapbase.html");
   QString fileName;
   QUrl baseUrl;
-  if (QFile(baseFile).exists()) {
+  if (!baseFile.isEmpty()) {
```

Comment:
why not check existence?

Reply:
QStandardPaths::locate() "Returns the absolute path to the first file or directory found, otherwise returns an empty string" according to its documentation.  Checking for existence would be redundant.